src/store/channels-list/saga.test.ts

```diff
@@ -186,34 +134,14 @@ describe('channels list saga', () => {
       groupChannelType: '',
     };
 
-    const { storeState } = await expectSaga(fetchChannelsAndConversations)
-      .provide([
-        [
-          matchers.call(chat.get),
-          chatClient,
-        ],
-        [
-          matchers.call(
-            [
-              chatClient,
-              chatClient.getChannels,
-            ],
-            rootDomainId
-          ),
-          [channel],
-        ],
-        [
-          matchers.call([
-            chatClient,
-            chatClient.getConversations,
-          ]),
-          [conversation],
-        ],
-        [
-          matchers.call.fn(delay),
-          null,
-        ],
-      ])
+    const additionalProviders = [
+      [matchers.call(chat.get), chatClient],
+      [matchers.call([chatClient, chatClient.getChannels], rootDomainId), [channel]],
+      [matchers.call([chatClient, chatClient.getConversations]), [conversation]],
+      [matchers.call.fn(delay), null],
+    ];
+
+    const { storeState } = await subject(fetchChannelsAndConversations, {}, additionalProviders)
```

Comment:
```suggestion
    const { storeState } = await subject(fetchChannelsAndConversations, {})
         .provide([
             [matchers.call([chatClient, chatClient.getChannels], rootDomainId), [channel]],
             [matchers.call([chatClient, chatClient.getConversations]), [conversation]],
             [matchers.call.fn(delay), null],
         ])
```

The point of the wrapped expectSaga is to allow multiple calls to provide. That's what allows us to set up default providers in `subject` and then "add on".